code/modules/clothing/masks/gas_filter.dm

```diff
@@ -0,0 +1,100 @@
+///List of gases with high filter priority
+GLOBAL_LIST_INIT(high_filtering_gases, list(
+		/datum/gas/plasma,
+		/datum/gas/carbon_dioxide,
+		/datum/gas/nitrous_oxide
+	))
```

Comment:
```suggestion
	/datum/gas/plasma,
	/datum/gas/carbon_dioxide,
	/datum/gas/nitrous_oxide
))
```

As a rule of thumb, indenting twice is always wrong.

Do it here and everywhere else.